elastic/kibana
# [Synthetics UI] Update synthetics-monitor saved object name normalizer

## Summary

Fixes #139301.

Updates synthetic monitor object type to perform case-insensitive sorting on retrieval.


### Checklist

Delete any items that are not applicable to this PR.


### How to test this PR?

- Create an empty oblt cluster and connect Kibana locally.
- Create three monitors in whatever order with the following names: 'A', 'B' and 'aa'.
- Navigate to Monitor Management monitor list and check that monitors are displayed in the following order: 'A' -> 'aa' -> 'B'.
- Enable Synthetics UI and check monitors are displayed in the same order.
